vars/all_var.yml

```diff
@@ -273,97 +111,21 @@
   app_fhir_cert_name: "{{ env_app_fhir_cert_name }}"
   app_fhir_key_name: "{{ env_app_fhir_key_name }}"
 
-  # cf_app_selinux_policy sets SELINUXTYPE=targeted
-  cf_app_selinux_policy: "{{ selinux_policy }}"
-  # cf_app_selinux_state set SELINUX=permissive
-  cf_app_selinux_state: "{{ selinux_state }}"
-
-  cf_app_sg_app_zone_id: "{{ env_cf_app_sg_app_zone_id }}"
-  cf_app_azone: "{{env_cf_app_azone }}"
-  cf_app_build_image_id: "{{ env_cf_app_build_image_id }}"
-  cf_app_instance_name: "{{ env_cf_app_instance_name }}"
-  cf_app_tag_key_stack: "{{ env_cf_app_tag_key_stack }}"
+  # legacy values still used in roles, should be renamed for clarity
   cf_app_tag_key_layer: "{{ env_cf_app_tag_key_layer }}"
-  cf_app_key_name: "{{ env_cf_app_key_name }}"
-  cf_app_instance_type: "{{ env_cf_app_instance_type }}"
-  cf_app_public_server_name: "{{ env_cf_app_public_server_name }}"
-  cf_app_pyserver_name: "{{ env_cf_app_pyserver_name }}"
-  cf_app_pyapp_settings: "{{ env_cf_app_pyapp_settings }}"
-  cf_app_repo_branch: "{{ env_cf_app_repo_branch }}"
-  cf_app_db_user: "{{ env_cf_app_db_user }}"
-  cf_app_db_pwd: "{{ env_cf_app_db_pwd }}"
-  cf_app_pyapps_email: "{{ env_cf_app_pyapps_email }}"
   cf_app_pyapps_pwd: "{{ env_cf_app_pyapps_pwd }}"
```

Comment:
Unfortunately these variables are still used in playbooks/roles, opted to leave it and not refactor for now.